prac_05/hex_colours.py

```diff
@@ -1,3 +1,11 @@
-CODE_TO_COLOUR = {"Red": "#ff0000", "Green": "#00ff00", "Blue": "#0000ff", "Orange": "#ffa500",
-                  "Yellow": "#ffff00", "White": "#ffffff", "Pink": "#ffb5c5", "Purple": "#9b30ff",
-                  "Black": "#000000", "Aqua": "#00ffff"}
+CODE_TO_COLOUR = {"red": "#ff0000", "green": "#00ff00", "blue": "#0000ff", "orange": "#ffa500",
+                  "yellow": "#ffff00", "white": "#ffffff", "pink": "#ffb5c5", "purple": "#9b30ff",
+                  "black": "#000000", "aqua": "#00ffff"}
+
+colour = input("Enter a colour: ").lower()
+while colour != "":
+    try:
+        print(colour, "is", CODE_TO_COLOUR[colour])
+    except KeyError:
+        print("Invalid colour name")
+    colour = input("Enter a colour: ").lower()
```

Comment:
Looks good.

Reply:
Good work using EAFP hear, looking back now, I should have done this it makes more sense. 👍